lib/takeaway.rb

```diff
@@ -0,0 +1,16 @@
+class Takeaway
+  attr_accessor :order
+  attr_reader :menu_ready
+  def initialize menu
```

Comment:
Unused method argument - `menu`. If it's necessary, use `_` or `_menu` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.